spec.bs

```diff
@@ -185,44 +185,59 @@ Each {{URLPattern}} object has an associated <dfn for=URLPattern>search componen
 Each {{URLPattern}} object has an associated <dfn for=URLPattern>hash component</dfn>, a [=component=], which must be set upon creation.
 
 <dl class="domintro non-normative">
-  <dt><code>{{URLPattern}} . {{URLPattern/protocol}}</code>
+  <dt><code>|urlPattern| = new {{URLPattern/constructor(input, baseURL)|URLPattern}}(|input|, |baseURL|)</code></dt>
+  <dd class=allow-2119>
+    Constructs a new URLPattern object.  The |input| may be a {{URLPatternInit}} dictionary or a string.  If |input| is a string, then an optional |baseURL| string may also be provided.
+  </dd>
```

Comment:
I would suggest separate dt/dd pairs for the two overloads since they have rather different semantics and parameter orders. Even though they're not formal Web IDL overloads for the reasons discussed in #35, probably web developers should think of them as separate.

Same for test and exec.